GVFS/GVFS.Installer/CreateMacInstaller.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
```

Comment:
And to that point, have you confirmed that an error from this script will fail the csproj build?

Reply:
```BuildGVFSForMac.sh``` logs red error text in the Console and exits after Publish.

```
  GVFS.Installer.Mac -> /Users/Ameen/Work/GVFS/BuildOutput/GVFS.Installer.Mac/bin/x64/Debug/netcoreapp2.1/osx-x64/GVFS.Installer.Mac.dll
  GVFS.Installer.Mac -> /Users/Ameen/Work/GVFS/Publish/
  /Users/Ameen/Work/GVFS/VFSForGit/GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh Debug 0.2.173.2 /Users/Ameen/Work/GVFS/VFSForGit/../BuildOutput/GVFS.Installer.Mac/bin/x64/Debug/netcoreapp2.1/osx-x64/
  Build output directory not specified
/Users/Ameen/Work/GVFS/VFSForGit/GVFS/GVFS.Installer.Mac/GVFS.Installer.Mac.csproj(34,5): error MSB3073: The command "/Users/Ameen/Work/GVFS/VFSForGit/GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh Debug 0.2.173.2 /Users/Ameen/Work/GVFS/VFSForGit/../BuildOutput/GVFS.Installer.Mac/bin/x64/Debug/netcoreapp2.1/osx-x64/" exited with code 1.
```